packages/e2e-tests/src/specs/editor/shopping/woocommerce.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+
+// eslint-disable-next-line import/no-extraneous-dependencies -- in package.json
+import { matchers } from 'jest-json-schema';
```

Comment:
The dependency needs to be in the dependencies of `e2e-tests/package.json`, not the main `package.json`.

Then this lint warning will go away.

Reply:
moved https://github.com/GoogleForCreators/web-stories-wp/pull/11250/commits/93cf9536bf636c5e58b44bb9463e5c0217eb590e